pontoon/base/static/js/fluent_interface.js

```diff
@@ -140,7 +119,7 @@ var Pontoon = (function (my) {
 
       // Render SelectExpression
       if (element.expression && element.expression.variants) {
-        var expression = serializeExpression(element.expression.expression);
+        var expression = fluentSerializer.serializeExpression(element.expression.expression);
```

Comment:
Cool!

Reply:
The `Cool` was about using the upstream `serializeExpression`.

I do have another comment about `isSelectExpressionElement`: How is it better than the following? :)

```js
if (element.type === `SelectExpression`) {
```